### PR TITLE
Add mention for required bundle file when isntalling from gem

### PR DESCRIPTION
**Pull request by dgagnon**
Documentation for :

Missing file when installing from gem #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ for how to install Smart Proxy plugins
 
 This plugin is compatible with Smart Proxy 1.10 or higher.
 
+When installing using "gem", make sure to install the bundle file:
+echo "gem 'smart_proxy_dns_powerdns'" > /usr/share/foreman-proxy/bundler.d/dns_powerdns.rb
```
Comment (ekohl) [inline README.md]: If you leave an empty line between these two and indent the echo it formats nicely as code.

**Comment by ekohl** [comment]
If you could amend the current commit with the changes and include `Closes GH-6` then it should automatically close the issue when it's merged.

**Comment by dgagnon** [comment]
ekohl, done as requested !

**Comment by ekohl** [comment]
Thanks! I squashed the commits for you.

**Pull request closed without merging**